streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java

```diff
@@ -46,4 +52,27 @@
         store.init(context, store);
         return store;
     }
+
+    @Test
+    public void shouldPutAll() {
+        List<KeyValue<Integer, String>> entries = new ArrayList<>();
+        entries.add(new KeyValue<Integer, String>(1, "1"));
+        entries.add(new KeyValue<Integer, String>(2, "2"));
+        store.putAll(entries);
+        assertEquals(store.get(1), "1");
+        assertEquals(store.get(2), "2");
+    }
+
+    @Test
+    public void shouldCreatePersistentStore() {
```

Comment:
This test doesn't seem to belong here. The test class is `InMemoryKeyValyLoggedStoreTest`, yet the test is `shouldCreatePersistentStore` If anything this should be moved to `StoresTest`, but maybe it is already covered